packages/cardhost/app/components/cardhost-right-edge.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glimmer/component';
+
+const typeToInputTypeMap = {
+  '@cardstack/core-types::string': 'Text Field',
+  '@cardstack/core-types::case-insensitive': 'Text Field',
+  '@cardstack/core-types::boolean': 'Checkbox',
+  '@cardstack/core-types::date': 'Date',
+  '@cardstack/core-types::integer': 'Text Field',
+}
```

Comment:
this seems duplicative with what we have in the field renderer--please lets not redefine this. Rather let's figure out a way to only define this once.